detectable/src/main/java/com/blackduck/integration/detectable/detectables/opam/parse/OpamFileParser.java

```diff
@@ -105,6 +95,22 @@ public Map<String, String> parseData(List<String> lines) {
         return output;
     }
 
+    private void addProjectNameAndVersion(String line, Map<String, String> output, Pattern pattern, String parsingValue) {
+        String value = line.split(":")[1];
+        Matcher matcher = pattern.matcher(value);
+        if (matcher.find()) {
+            value = matcher.group(1);
+        }
+
+        if(parsingValue.equals(NAME)) {
```

Comment:
If `parsingValue` is always either name or version you can simply do `output.put(parsingValue, value)` and avoid the if statement.